spring-kafka/src/main/java/org/springframework/kafka/listener/KafkaMessageListenerContainer.java

```diff
@@ -896,6 +921,11 @@ private void commitIfNecessary() {
 			}
 		}
 
+		@Override
+		public void seek(String topic, int partition, long offset) {
+			this.seeks.add(new TopicPartitionInitialOffset(topic, partition, offset));
```

Comment:
My concern is that the topics/partitions can move when the broker does a rebalance of the assignment - I want to be sure that the seek is done in the context of the current state at the time the listener is called. That way, if the consumer no longer owns that partition the seek operation will fail.